docker-compose.yml

```
@@ -0,0 +1,94 @@
+    ports:
+      - "8000:8000"
+    networks:
+      - backend
+  
+  kafka-db-consumer:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: python -m backend.kafka.kafka_db_consumer
+    env_file:
+    - .env
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)